Vagrantfile

```diff
@@ -97,6 +97,11 @@ if File.exist?(env_specific_config_path)
   end
 end
 
+# Display the main current configuration parameters
+@ui.info "Welcome to Kubernetes playground !"
```

Comment:
Extra space after `playground`. Remove

Reply:
I added on purpose because graphically was more evident...
De gustibus non disputandum ... I'll leave the ui of k8s-pg to you :-)
I've removed it
I'll open an issue to propose:
1. Removal of the message about vagrant version
2. Adding the k8s-pg version number in the Welcome line